bin/read-only-file-differ.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+base_dir=$(cd $(dirname $0)/.. && pwd)
+
+VERSION=`grep 'curr.release' $base_dir/gradle.properties | sed -e 's/curr.release=\(.*\)/\1/'`
+echo "Voldemort version detected: $VERSION"
+
+# The jar file's name depends on the directory name...
+export VOLDEMORT_JAR="$(echo $base_dir/build/libs/voldemort-$VERSION.jar)"
+
+java -cp $VOLDEMORT_JAR:$base_dir/lib/* voldemort.store.readonly.ReadOnlyFileDiffer $@
```

Comment:
Did you take a look at the voldemort-shell.sh ? We have bin/run-class.sh for the same purpose. I don't see dependent jars in the class path.